llvm/utils/release/build_llvm_release.bat

```diff
@@ -106,8 +108,8 @@ echo Using VS devcmd: %vsdevcmd%
 :: start echoing what we do
 @echo on
 
-set python32_dir=C:\Users\%USERNAME%\AppData\Local\Programs\Python\Python310-32
-set python64_dir=C:\Users\%USERNAME%\AppData\Local\Programs\Python\Python310
+set python32_dir=C:\Users\%USERNAME%\AppData\Local\Programs\Python\Python311-32

```

Comment:
Hmm maybe we should just look for the `python` in `PATH`? Any reason why we hardcode this here?

Reply:
yeah I don't think this is great, but I think it's because you can't have 32 and 64 bit python in the path at the same time? I wonder if we should probe for this in some better way than hardcoding paths. cc @zmodem

Reply:
For how long do we want to keep releasing the x86 builds? At Ubisoft I removed Windows 32-bit support for our game years ago. Do you happen to have metrics in github to see if anybody actually downloads 32-bit LLVM packages?

Reply:
LLDB is super fiddly about the Python stuff, and it's great at finding the versions on my system which don't work. It's also important that the version used in the package is consistent.

In general, any probing is really bad for building reproducible packages. I think what we should really do is have the script download the exact Python versions we want.

But, for local testing it might make sense to bypass this. Maybe there could be another flag for that.

Reply:
Yeah downloading the right one and allowing a local override sounds like the right way forward.

Reply:
> LLDB is super fiddly about the Python stuff, and it's great at finding the versions on my system which don't work. It's also important that the version used in the package is consistent.

That probably explains the LLDB [test errors I'm seeing](https://github.com/llvm/llvm-project/issues/77992). I'll try with Python 3.10 see if that solves the issues.

> In general, any probing is really bad for building reproducible packages. I think what we should really do is have the script download the exact Python versions we want.

100% agree. Ideally we should do the same for all the other tools. In all the current/past compagnies I've always tried to make product compile on a clean VM, by pulling on specific versions. We could use `scoop` or `choco` for that purpose?

> But, for local testing it might make sense to bypass this. Maybe there could be another flag for that.

Ok, I can add that.